src/main/scanLogic/scanRunners/binaryRunner.ts

```diff
@@ -276,27 +276,45 @@ export abstract class BinaryRunner {
                     })
             );
         }
-        let hadError: boolean = false;
+        let exeErr: Error | undefined = undefined;
```

Comment:
Redundant assignment
```suggestion
        let exeErr: Error | undefined;
```